apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -101,4 +103,29 @@ describe('Delegation Controller', () => {
         revokedAt: '0x00000000',
       });
   });
+
+  it('(POST) /v1/delegation/revokeDelegation/:accountId/:providerId', async () => {
+    const providerId = provider.msaId?.toString();
+    const { keypair } = users[1];
+    const accountId = keypair.address;
+    const getPath: string = `/v1/delegation/revokeDelegation/${accountId}/${providerId}`;
+    const response = await request(app.getHttpServer()).get(getPath).expect(200);
```

Comment:
Could either:
* add a separate test for `(GET) /v1/delegation/revokeDelegation/:accountId/:providerId` specifically, -OR-
* add some `expect` statements here to validate the response payload, and continue on to POST